client/packages/common/src/ui/components/buttons/standard/BaseButton.tsx

```diff
@@ -44,30 +44,30 @@ export const StyledBaseButton = styled(MuiButton)(({
   const iconColor = getIconColor();
 
   return {
+    '&.MuiButton-contained': { fontWeight: 600 },
     '&.MuiButton-outlined': {
       backgroundColor: 'white',
-      color: color === 'primary' ? 'black' : color,
+      color:
+        color === 'primary'
+          ? theme.mixins.button.textColor
+          : translateColor(theme, color),
       '& .MuiButton-startIcon': {
-        color: color === 'primary' ? translateColor(theme, color) : color,
+        color: color === 'primary' ? translateColor(theme, color) : undefined,
```

Comment:
the `color` here is 'primary' | 'secondary' etc and therefore not a valid colour. The effect and intention is the same as setting undefined